java/code/src/com/redhat/rhn/taskomatic/task/payg/PaygUpdateHostsTask.java

```diff
@@ -64,8 +64,17 @@ private void loadHttpsCertificates(List<CloudRmtHost> hostToUpdate) throws JobEx
             log.error("error when writing the hosts file", e);
         }
         finally {
-            String[] cmd = {"/usr/share/rhn/certs/update-ca-cert-trust.sh"};
-            executeExtCmd(cmd);
+            if (hostToUpdate.size() > 0) {
+                try {
+                    String[] cmd = {"systemctl", "is-active", "--quiet", "ca-certificates.path"};
+                    executeExtCmd(cmd);
+                }
+                catch (Exception e) {
+                    log.debug("ca-certificates.path service is not active, we will call 'update-ca-certificates' tool");
+                    String[] cmd = {"/usr/sbin/update-ca-certificates"};
```

Comment:
Can we call `/usr/share/rhn/certs/update-ca-cert-trust.sh` instead?
As you know the community is trying to port this to EL8 which use a different script for the same thing.
This bash script should make the difference between SUSE and EL.

Reply:
Sure Micheal, I have changed the call to point to the script. Thanks